src/imio/smartweb/core/contents/rest/news/endpoint.py

```
@@ -51,21 +52,16 @@ def query_url(self):
-        #     item = [k for k, v in data.items() if "administration" in v.lower()][0]
-        #     if not item:
-        #         selected_item = uids[0]
-        #     selected_item = item
-
-        # UNDO CACHE
-        # "selected_news_folders={}".format(selected_item),
-        # f"entity_uid={entity_uid}",
+        uids, data = self._get_news_folders_uids_and_title_from_entity(entity_uid)
+        selected_item = self.context.selected_news_folder
+        if selected_item not in uids:
+            item = [k for k, v in data.items() if "administration" in v.lower()][0]
```

Ruff (0.15.2)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)